src/Components/ReviewCard.jsx

```diff
@@ -0,0 +1,23 @@
+export default function ReviewCard ({reviews}) {
```

Comment:
`Card` is a design term for a single item that looks like a card. As such this component should just be a single `li` rather than the whole list. 

If you want to render the whole list in here then rename it to something more appropriate, either way is good 👍